File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonPdf.java
package com.zawzaw.savethelibrary.model.gson;

import com.google.gson.annotations.SerializedName;

/**
 * Created by zawzaw on 04/01/18.
 */

public class GsonPdf {

    @SerializedName("pdf_id")
    private int pdf_id;

    @SerializedName("pdf_title")
    private String pdf_title;

    @SerializedName("download_link")
    private String download_link;

    @SerializedName("image")
    private String pdf_image;

    @SerializedName("category_mm")
    private String category_mm;

    @SerializedName("category_en")
    private String category_en;

    @SerializedName("pdf_source")
    private String pdf_source;

    public int getPdf_id() {
        return pdf_id;
    }

    public String getPdf_title() {
        return pdf_title;
    }

    public String getDownload_link() {
        return download_link;
    }

    public String getPdf_image() {
        return pdf_image;
    }

    public String getCategory_mm() {
        return category_mm;
    }

    public String getCategory_en() {
        return category_en;
    }

    public String getPdf_source() {
        return pdf_source;
    }

}

File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonPdf.kt
package com.zawzaw.savethelibrary.model.gson

import com.google.gson.annotations.SerializedName

/**
 * Created by zawzaw on 04/01/18.
 */

class GsonPdf {

    @SerializedName("pdf_id")
    val pdf_id: Int = 0

    @SerializedName("pdf_title")
    val pdf_title: String? = null

    @SerializedName("download_link")
    val download_link: String? = null

    @SerializedName("image")
    val pdf_image: String? = null

    @SerializedName("category_mm")
    val category_mm: String? = null

    @SerializedName("category_en")
    val category_en: String? = null

    @SerializedName("pdf_source")
    val pdf_source: String? = null

}
